Patterns/English/English-Numbers.yaml

```diff
@@ -23,19 +23,25 @@ PlaceHolderDefault: !simpleRegex
   def: \D|\b
 NumbersWithPlaceHolder: !paramsRegex
   def: (((?<!\d+\s*)-\s*)|(?<=\b))\d+(?!(\.\d+[a-zA-Z]))(?={placeholder})
+  def_js: (((?!\d+\s*)-\s*)|(?=\b))\d+(?!(\.\d+[a-zA-Z]))(?={placeholder})
```

Comment:
Is there an easy or suggested way to convert or test .net regexes to .js?
It would be nice to have a readme file in the repo that gives guidelines or pointers to new devs on how to do it and what to pay attention to.